docs/microsoft-365-developer-program-faq.yml

```diff
@@ -188,12 +190,12 @@ sections:
       - question: |
           Can I share my developer subscription with my team?
         answer: |
-          As a member of the Microsoft 365 Developer Program, you are entitled to one free Microsoft 365 developer subscription. Your subscription is associated with your developer program account. You can't transfer owership of your subscription to a different account.  
+          Qualified members of the Microsoft 365 Developer Program are entitled to one Microsoft 365 E5 developer subscription. If you qualify for a subscription, your subscription is associated with your developer program account. You can't transfer owership of your subscription to a different account.  
       
       - question: |
           Why don't you offer a one-year subscription?
         answer: |
-          In April 2019, we transitioned to a new model where your subscription can be renewed perpetually every 90 days if you're actively using it for development. We believe that this model ensures that developers who are actively developing solutions have a subscription for as long as they need one. If you're developing frequently, your subscription never expires; it is automatically extended. If you're developing for a short time, and your subscription expires and is deleted, you can sign up for a new one. 
+          In April 2019, we transitioned to a new model where your subscription can be renewed perpetually every 90 days if you're actively using it for development. We believe that this model ensures that developers who are actively developing solutions have a subscription for as long as they need one. If you're developing frequently, your subscription never expires; it is automatically extended. If you qualify for a subscription and you're developing for a short time, and your subscription expires due to inactivity and is deleted, you can set up a new one. 
```

Comment:
Question: Does the subscription expire now? This will pertain to current members, not new?

Reply:
Yes, this applies to current members who didn't get an autorenewing subscription through Visual Studio.